config/logging.yaml

```diff
@@ -53,7 +55,7 @@
   :log_trace: false
   :level: info
   :type: file
-  :pattern: "%d [%c] [%.1l] %m\n"
+  :pattern: "%d %.8X{session} [%c] [%.1l] %m\n"
```

Comment:
What's the benefit of having the session here vs. request id? The issue here is with user performing multiple requests at the same time. Also using the session id as a shared UI between other services that are called as part of the request is limiting, as you can't group the log messages by user requests.